### PR TITLE
[MM-29172] Add E2E tests for teamUsersRemoveCmdF

### PR DESCRIPTION
**Pull request by jasimmons**
#### Summary

Adds end-to-end tests for `teamUsersRemoveCmdF`.

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/15678

### CODE REVIEW

**Review by mgdelacroix — APPROVED** [review]
LGTM! :+1: Thanks for the PR @jasimmons!!

File: commands/team_users_e2e_test.go [hunk 1]
```diff
@@ -159,3 +159,54 @@ func (s *MmctlE2ETestSuite) TestTeamUserAddCmd() {
...
+
+		err := teamUsersRemoveCmdF(c, &cobra.Command{}, []string{team.Name, user.Username})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 0)
+		s.Require().Len(printer.GetErrorLines(), 0)
+
+		teamMembers, err := s.th.App.GetTeamMembers(team.Id, 0, 10, nil)
+		s.Require().Nil(err)
+		s.Require().NotNil(teamMembers)
+		s.Require().Len(teamMembers, 0)
```
Comment (ashishbhate) [inline commands/team_users_e2e_test.go:189]: You'll need to move the user creation and addition to team inside the test. `RunForSystemAdminAndLocal` runs the test function twice. First as SystemAdmin and second time as Local Client.

The SystemAdmin test run removes the user successfully. For the local client test run the user has already been removed from the team so it always passes. mmctl doesn't complain if a user doesn't already exist.

**Review by ashishbhate — COMMENTED** [review]
Thanks @jasimmons! Looking good. Just one suggested fix inline.

File: commands/team_users_e2e_test.go (hunk 1 above)
Comment (jasimmons) [reply commands/team_users_e2e_test.go]: Thanks @ashishbhate, good catch! I moved it and committed in a `fixup` which I can rebase if/when you're 👍 on it.

**Review by ashishbhate — APPROVED** [review]
Thanks @jasimmons! LGTM

**Review by lindy65 — APPROVED** [review]
Thanks @jasimmons @mgdelacroix - looks good to me 👍

**Pull request merged**